### DISCUSSION TITLE
HandMenu prefabs not respecting themes?

### DISCUSSION THREAD

**Discussion by bjornsyse** [microsoft/MixedRealityToolkit-Unity#11116] [Q&A] [answered]
Hi

I'm in the `CoreUXThemingExample `Scene and I've added a few of the hand menu prefabs. It seems only the first empty one honors the theming system, is that right? 

![Unity_HBZxnEJs19](https://user-images.githubusercontent.com/26255606/196677855-c91f78cd-f065-4fe5-b038-0365b1fbc6f8.gif)

I tried adding `UXBindingConfigurator` to them but to no avail. Is this because they're using the canvas system? They are placed in the non-canvas path:

![image](https://user-images.githubusercontent.com/26255606/196678781-e1354712-fe8b-42f2-87be-2d3ebbf36b47.png)

Best regards

**Comment by Zee2** [comment, marked as answer]
Hi there,

The theming system is still in early preview and is subject to change. We're working on shaping our theming story moving forward, and the Canvas-based UI prefabs don't currently have their theming systems hooked up.

**Reply by bjornsyse** [reply]
Ok, Figured it was something like that. Looking forward to your progress!
